patches/chrome-android-java-res-layout-bottom_toolbar_browsing.xml.patch

```diff
@@ -1,20 +1,50 @@
 diff --git a/chrome/android/java/res/layout/bottom_toolbar_browsing.xml b/chrome/android/java/res/layout/bottom_toolbar_browsing.xml
-index 76d517641db816c12bd35ae6497cd172eea71fd0..e56cffeee66034c659ac7719899f2512a1851c72 100644
+index 76d517641db816c12bd35ae6497cd172eea71fd0..2d13046d3bdac0e21bb9032d9358728b917ec95a 100644
 --- a/chrome/android/java/res/layout/bottom_toolbar_browsing.xml
```

Comment:
do we care about continuing to track upstream changes to this file?

Reply:
As to me the divergence with chromium is not that big, as of today, to stop tracking it. But it seems like chromium has different vision on how to use toolbars. So some day we will probably have to move on our own implementation. Personally I think it's still worth to track it.
@bbondy @SergeyZhukovsky wdyt?

Reply:
So what is the final decision on this?

Reply:
I would keep it for now.